Controllers/ExpenseController.cs

```diff
@@ -1,30 +1,62 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
+using System.Data.SqlClient;
+using Microsoft.Extensions.Configuration;
 
 namespace ExpensesWebApp.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
+
     public class ExpenseController : ControllerBase
+
     {
         private readonly ILogger<ExpenseController> _logger;
+        public IConfiguration Configuration { get; }
 
-        public ExpenseController (ILogger<ExpenseController> logger)
+        public ExpenseController (ILogger<ExpenseController> logger, IConfiguration configuration)
         {
             _logger = logger;
+            Configuration = configuration;
         }
-
+        
         [HttpGet]
-        public IEnumerable<Expense> Get()
+        public IEnumerable<ExpenseItem> Get()
         {
-            return Enumerable.Range(1, 1).Select(index => new Expense
+            return Enumerable.Range(1, 1).Select(index => new ExpenseItem
             {
-                ExpenseSum = 10
+                Expenses = 10
             })
             .ToArray();
         }
+        [HttpPost]
+        public void Post()
+        {
+            string connStr = Configuration.GetConnectionString("DatabaseConnectionString");

```

Comment:
Duombazės connection kūrimą iškeliam į atskirą klasę, nes kodą dubliuot negerai:
https://www.softwareyoga.com/is-your-code-dry-or-wet/
Ir ta klasė turėtų ateit per DependencyInjection:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-3.1